src/Assignment1/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+
+namespace Assignment1
+{
+    /// <summary>
+    /// This is main class <see cref="Program"/>
```

Comment:
```suggestion
    /// Main class <see cref="Program"/>
```

Throughout the code, I see the word **this** is used for describing the class and method in the documentation. Instead do similar to the suggestion